core/arch/asm_defines.asm

```diff
@@ -910,27 +910,38 @@ ASSUME fs:_DATA @N@\
         mov      ARG1, p1   @N@\
         blx      callee
 #elif defined(RISCV64)
+/* For RISC-V, there is not a instruction could operate both immediate
+ * and registers. Here is a macro which to judge if the arguments is
+ * registers or not.
+ */
+.set register.sp, 1
```

Comment:
So we need a whole bunch more of these.  Always include a `TODO i#3544: ...` comment in unfinished code so the reader knows it's not complete.

Reply:
Done